### ISSUE TITLE
listItemType in Fusion Layer Properties

### ISSUE THREAD

**Issue by rentonxp** [google/earthenterprise#1018]
Hello,

How to prevent all items from being made visible at once in a New layer group (checkOffOnly) ?
I do not find it in Fusion Layer Properties. 
Thx

Xavier

**Comment by rentonxp** [comment]
It would be a great improvement to add this function in Fusion "Layer Group Properties" option.

**Comment by tst-lsavoie** [comment]
Sorry for the slow movement on this. I'm not sure I understand the question, but if I understand you correctly and you don't want to show a layer immediately when you log in to the server, you can set the "Initial State" to Off in "Layer Properties". You get to that dialog box by opening a project, right clicking a resource, and clicking "Layer Properties".

![layerproperties](https://user-images.githubusercontent.com/9662346/48667143-28c3a980-ea8d-11e8-92e4-a3da905adbd5.png)

Let me know if that's not what you're asking about.

**Comment by rentonxp** [comment]
Not exactly. I'm trying to find this function in fusion (kml reference) :
checkOffOnly - When specified for a Container or Network Link, prevents all items from being made visible at once—that is, the user can turn everything in the Container or Network Link off but cannot turn everything on at the same time. This setting is useful for Containers or Network Links containing large amounts of data.

Thx
